QuantConnect.DataBento/DataBentoRawLiveClient.cs

```diff
@@ -95,27 +94,27 @@ public async Task<bool> ConnectAsync()
                 var port = parts.Length > 1 ? int.Parse(parts[1]) : 13000;
```

Comment:
- We can split the host and port in the constructor and reuse it from the global scope.
- when `parts.Length == 1`? The` _gateway` is always the same.